terraform/cloudfront/instances.tf

```diff
@@ -31,3 +31,16 @@ module "cloudfront_paas_docs" {
   system_dns_zone_id    = "${var.system_dns_zone_id}"
   system_domain_cert_id = "${var.system_domain_cert_id}"
 }
+
+module "redirect_paas_product_page" {
+  source = "./cloudfront_redirect"
+
+  name = "paas-product-page"
```

Comment:
It's probably worth including the `DEPLOY_ENV` in here so we can distinguish them in the AWS console.